src/View/Helper/DateHelper.php

```diff
@@ -124,7 +124,7 @@ public function ago($date, $alone=true)
         $diff = Time::fromNow($dateObj);
 
         if ($diff->days > 30) {
-            $formattedDate = $dateObj->nice();
+            $formattedDate = $dateObj->i18nFormat([\IntlDateFormatter::LONG, \IntlDateFormatter::SHORT]);
```

Comment:
This code results in the following:
![image](https://user-images.githubusercontent.com/5107734/89697882-8d3c3180-d91e-11ea-98a5-ececc5461616.png)

I think you could as well remove the time:
```
$formattedDate =  $dateObj->i18nFormat([\IntlDateFormatter::LONG, \IntlDateFormatter::NONE]);
```
So that we get instead:
![image](https://user-images.githubusercontent.com/5107734/89697919-ba88df80-d91e-11ea-925d-5a20d3644d09.png)


The rationale is that we are not displaying the time for dates between 1 and 30 days ago, so we shouldn’t display it for dates older than 30 days ago too.

Reply:
that makes sense!